src/server.ts

```diff
@@ -23,7 +23,7 @@ const exposeDocs = true
     })
 
     try {
-      await adminApp.listen({ port: 5001, host: '0.0.0.0' })
+      await adminApp.listen({ port, host })
```

Comment:
The port for the admin app should be different from the app port,
we can have a different env for this ex: `ADMIN_PORT`

Reply:
you can simply default it to `5001` for backward compatibility

```
Number(getOptionalConfigFromEnv('ADMIN_PORT')) || 5001
```